### ISSUE TITLE
dash update gets installed sooner than glibc which breaks updates from 20221001 images

### ISSUE THREAD

**Issue by chloris-pale-green** [void-linux/void-packages#41799]
I hope I am posting this in the right place. I've tried to set up a fresh install of Void on Raspberry Pi 4 (8 GB RAM, 64 GB MMC), but both the image method and the rootfs method resulted in the same issue: the installation was unable to upgrade.

When performing the first upgrade using `xbps-install -Su`, the following shows up (had to type this to my main PC, might contain some errors):

```text
diffutils-3.8_1: updating to 3.9_1 ...
/bin/sh: /usr/lib64/libc.so.6: version 'GLIBC_2.33' not found (required by /bin/sh)
/bin/sh: /usr/lib64/libc.so.6: version 'GLIBC_2.34' not found (required by /bin/sh)
```

I think it has nothing to do with `diffutils`, since very similar error pops up whenever I try to run `/bin/sh` or `dash` afterwards. The system did not show any graphic output after rebooting either (suggesting a broken shell?).

Is it possible that the image and tarball have to be rebuilt with correct Glibc versions?

**Comment by paper42** [comment]
fixed in 576382c2bbe3e72467842ee36b3e7264d49c2e8c

**Comment by chloris-pale-green** [comment]
@paper42, I've performed another test on a fresh install and xbps insisted on updating itself before doing anything else. `xbps-install -u xbps` pulled in the new `glibc` as a dependency and everything was fine afterwards. Thanks for the fix.
